modules/apps_api/app/workers/apps_api/fetch_connections.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require_relative '../../../lib/apps_api/notification_service.rb'
+
+module AppsApi
+  class FetchConnections
+    include Sidekiq::Worker
+    include SentryLogging
+
+    def perform
+      notif_service = AppsApi::NotificationService.new
+      # process all connection events
+      notif_service.handle_event(
+        'app.oauth2.as.consent.grant',
+        Settings.vanotify.template_id.connection_template
+      )
+      # process all disconnection events
+      notif_service.handle_event(
+        'app.oauth2.as.token.revoke',
+        Settings.vanotify.template_id.disconnection_template
```

Comment:
This should be `Settings.vanotify.lighthouse.template_id.connection_template`
or `Settings.vanotify.lighthouse.templates.connection_template`

@saman-moshafi-va where are you with the changes to settings yaml structure?

Reply:
Based on Saman's response here, I've just updated the Settings.yml and test.yml as well. Latest commit has the `Settings.vanotify.services.lighthouse.template_id.disconnection_template` naming convention added in his PR (https://github.com/department-of-veterans-affairs/vets-api/pull/5998/files/767be6e83acb9d3b3d3fe318b7c26ee597f9140f)

Reply:
[Waiting for the PR 5998 approval](https://github.com/department-of-veterans-affairs/vets-api/pull/5998).

Reply:
I'll leave the changes to settings.yml up to your PR then